src/core/client/auth/views/SignIn/SignInWithEmail.tsx

```diff
@@ -50,7 +50,7 @@ const SignInWithEmail: FunctionComponent<SignInWithEmailForm> = (props) => {
               />
             )}
             <div className={cn(CLASSES.login.field, styles.field)}>
-              <EmailField disabled={submitting} autofocus />
+              <EmailField disabled={submitting} />
```

Comment:
If you follow @cvle's suggestion and remove `dirty` from the `hasError` check, you can put this `autofocus` back in which improves keyboard usability by allowing folks to immediately start typing in the form when logging in with email.

Reply:
Thanks @nick-funk  It thought I put out a similar comment, but I guess I didn't click on submit :-P

Reply:
ah i forgot to re-add that